rotatescreen/mac_display.py

```diff
@@ -0,0 +1,91 @@
+"""Implementation rotation for Mac-Os"""
+import Quartz
+import objc
+import Foundation
+import warnings
+import platform
+
+__all__ = ['Display', 'get_displays']
```

Comment:
I didn't understand behaviour this methods((

Reply:
`get_primary_display` should return a `<rotatescreen Display>` object for the primary display, i.e. the one identified by the system as the user's main display. For example, this may be the attached display on a laptop. `get_secondary_displays` should return a list of `<rotatescreen Display>` objects for all other displays excluding the primary display.

Reply:
I will try add ``get_primary_display`` and ``get_secondary_display``